### DISCUSSION TITLE
Share Button for Podcasts or Media

### DISCUSSION THREAD

**Discussion by joelpoc** [AzuraCast/AzuraCast#7583] [Ideas]
Hello, and thank you for this amazing project!

I have a question or perhaps a feature request. Is there any way to enable a button or functionality to share individual podcasts or media on social networks or at least copy a direct link to them? I’ve been exploring the platform, but I haven’t been able to find this feature, and I’m not sure if I’ve missed it or if it’s not currently available.

This would be a very useful feature, as right now, if you want to share a specific podcast, you have to share the entire page and instruct others to look for it manually. Having a direct sharing option would greatly improve user experience.

Thank you for considering this!

**Comment by BusterNeece** [comment]
Hello! If you don't mind, can you submit this as a feature request over on our FeatureUpvote site at https://features.azuracast.com/? Thank you!

**Reply by joelpoc** [reply]
Sure! Thanks.

**Comment by joelpoc** [comment]
Hi Buster,

A week has passed, and it still says 'Awaiting approval.' Is there anything
else I need to do?

https://features.azuracast.com/suggestions/596275/share-button-for-podcasts-or-media

Best,
Joel

**Reply by BusterNeece** [reply]
@joelpoc Nothing else you need to do. Patience!
